mecab/Rakefile

```diff
@@ -30,9 +30,34 @@ class MecabPackageTask < PackagesGroongaOrgPackageTask
 
   def yum_targets_default
     [
-      "almalinux-8"
+      "almalinux-8",
+      "centos-8"
     ]
   end
+
+  def detect_srpm_base_url
+    base_url = ""
+    yum_targets.each do |target|
+      if target == "almalinux-8"
+        base_url = "https://repo.almalinux.org/almalinux/8.4/AppStream/Source/Packages"
+      else
+        base_url = "https://vault.centos.org/8.5.2111/AppStream/Source/SPackages"
+      end
+    end
+    base_url
```

Comment:
It seems that this doesn't work without `YUM_TARGETS=almalinux-8` and `YUM_TARGETS=centos-8`.